apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.jsx

```diff
@@ -183,6 +215,15 @@ class ProgressTableContainer extends React.Component {
     this.setState({rows});
   }
 
+  numExtraRowsWhenExpanded() {
```

Comment:
to me this name creates some ambiguity around whether this will return the number of extra rows per student, or the total number of extra rows currently in the state. i realize that `numDetailRows` isn't any better though. what do you think of `numDetailRowsPerStudent`?

Reply:
oh haha if i read the implementation it's pretty obvious given `const rowsPerStudent`... but i think it would still be helpful to surface that in the name